docs/standard/base-types/miscellaneous-constructs-in-regular-expressions.md

```diff
@@ -57,16 +57,16 @@ Regular expressions in .NET include three miscellaneous language constructs. One
 ## Inline Comment  
  The `(?#` *comment*`)` construct lets you include an inline comment in a regular expression. The regular expression engine does not use any part of the comment in pattern matching, although the comment is included in the string that is returned by the <xref:System.Text.RegularExpressions.Regex.ToString%2A?displayProperty=nameWithType> method. The comment ends at the first closing parenthesis.  
   
- The following example repeats the first regular expression pattern from the example in the previous section. It adds two inline comments to the regular expression to indicate whether the comparison is case-sensitive. The regular expression pattern, `\b((?# case-sensitive comparison)D\w+)\s((?#case-insensitive comparison)d\w+)\b`, is defined as follows.  
+ The following example repeats the first regular expression pattern from the example in the previous section. It adds two inline comments to the regular expression to indicate whether the comparison is case-sensitive. The regular expression pattern, `\b((?# case sensitive comparison)D\w+)\s(?ixn)((?#case insensitive comparison)d\w+)\b`, is defined as follows.  
   
 |Pattern|Description|  
 |-------------|-----------------|  
 |`\b`|Start at a word boundary.|  
-|`(?# case-sensitive comparison)`|A comment. It does not affect pattern-matching behavior.|  
+|`(?# case sensitive comparison)`|A comment. It does not affect pattern-matching behavior.|  
 |`(D\w+)`|Match a capital "D" followed by one or more word characters. This is the first capturing group.|  
 |`\s`|Match a white-space character.|  
 |`(?ixn)`|From this point on, make comparisons case-insensitive, make only explicit captures, and ignore white space in the regular expression pattern.|  
-|`(?#case-insensitive comparison)`|A comment. It does not affect pattern-matching behavior.|  
```

Comment:
The original line is correct. The comment in the regex (and in the element-by-element description of the regex pattern) should have been "case-insensitive comparison".